### ISSUE TITLE
java.lang.UnsatisfiedLinkError: could not load FFI provider com.spotify.docker.client.shaded.jnr.ffi.provider.jffi.Provider When building docker images

### ISSUE THREAD

**Issue by cerveada** [AbsaOSS/spline#1140]
Scala 2.12
Java 11
CPU M1
OS Mac OS

```
mvn install -Ddocker -Ddockerfile.repositoryUrl=my -DskipTests
```
[maven-docker-issue.log](https://github.com/AbsaOSS/spline/files/10112286/maven-docker-issue.log)

**Comment by cerveada** [comment]
Currently used docker maven plugin is not maintained anymore if the issue is not fixable possible replacement is:
 https://github.com/fabric8io/docker-maven-plugin

**Comment by wajda** [comment]
we could test it in this repo, but the fix should go to the [root-pom](https://github.com/AbsaOSS/root-pom) repo

**Comment by cerveada** [comment]
Workaround assuming current DOCKER_HOST is `unix:///Users/abac720/.colima/docker.sock` :
```bash
brew install socat
socat TCP-LISTEN:2375,range=127.0.0.1/32,reuseaddr,fork unix:///Users/abac720/.colima/docker.sock

# in different tab
export DOCKER_HOST=tcp://127.0.0.1:2375
mvn install -Ddocker -Ddockerfile.repositoryUrl=my -DskipTests
```
from here
https://stackoverflow.com/questions/71300031/docker-image-build-failed-on-mac-m1-chip

**Comment by wajda** [comment]
Fixed in AbsaOSS/root-pom#6 and #1166 respectively.
